### PR TITLE
E2E: The user MUST be able to see a list of all their accounts

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

Continuing with the accounts end-to-end test, we check the next requirement.

# Changes

1. Check the list of account names in the e2e test.
2. Required page object changes.
3. (not really related but somehow started causing trouble now) While signing in, make sure that we click the correct "Next" button otherwise we might click the same one a second time and get stuck on the next one.

# Tests

`npm run test-e2e`

### CODE REVIEW

File: frontend/src/tests/e2e/accounts.spec.ts [hunk 1]
```diff
@@ -23,9 +23,11 @@ test("Test accounts requirements", async ({ page, context }) => {
   const subAccountName = "My second account";
   await nnsAccountsPo.addAccount(subAccountName);
 
-  // TODO:
+  // AU001: The user MUST be able to see a list of all their accounts
+  const accountNames = await nnsAccountsPo.getAccountNames();
+  expect(accountNames).toEqual([mainAccountName, subAccountName]);
 
-  // AU001:  The user MUST be able to see a list of all their accounts
+  // TODO:
```
Comment (lmuntaner) [inline frontend/src/tests/e2e/accounts.spec.ts:30]: Why this `TODO`?

**Review by lmuntaner — APPROVED** [review]
Thanks!

File: frontend/src/tests/e2e/accounts.spec.ts (hunk 1 above)
Comment (dskloetd) [reply frontend/src/tests/e2e/accounts.spec.ts:30]: AU004 and AU005 below still must be done.
This TODO was on line 26 before but AU001 is now done so I moved the TODO down a bit more.

Comment (lmuntaner) [reply frontend/src/tests/e2e/accounts.spec.ts:30]: ok, understood, thanks!

**Pull request merged**